### ISSUE TITLE
EPANET incorrectly reads valve setting when its fixed status is OPEN or CLOSED not 'NONE'

### ISSUE THREAD

**Issue by Mariosmsk** [OpenWaterAnalytics/EPANET#583]
Repost from the issue here https://github.com/OpenWaterAnalytics/EPANET-Matlab-Toolkit/issues/168 by @stephen-young 

networks: https://github.com/OpenWaterAnalytics/EPANET-Matlab-Toolkit/files/4171367/valve_issue_networks.zip

I think this is an EPANET bug.

https://github.com/OpenWaterAnalytics/EPANET/blob/f18152482ce3833eaae35b81a01d08b6be4bed92/src/input3.c#L2131

Correct
`link_index = 6`
```
ENopen('simple_network_1P1T1D1V_valve_status_none.inp')
ENgetlinkvalue(link_index, EN_INITSETTING)

```

Incorrect
```
ENopen('simple_network_1P1T1D1V_valve_status_open.inp');
ENgetlinkvalue(link_index, EN_INITSETTING)
```

**Comment by eladsal** [comment]
It seems the function returns `MISSING`.

**Comment by LRossman** [comment]
This is a consequence of the original code design that made a valve's setting (`Link[i].Kc`) hold both the  valve's pressure/flow setting when it was active and a `MISSING `value to indicate it had a fixed `OPEN/CLOSED` status (my bad). An additional Valve variable should have been used to indicate whether the valve should operate in a fixed `OPEN/CLOSED` state or allow its setting to determine its actual state (such as when a PRV would be `OPEN `if its downstream pressure was below its setting).

Short of introducing this new Valve variable (which would impact a good chunk of code) the simplest fix is to have `ENgetlinkvalue(link_index, EN_INITSETTING)` return 0 if the setting equals `MISSING `which is what happens when `EN_SETTING` is retrieved for a fixed status valve.

**Comment by eladsal** [comment]
@LRossman so it is a feature, not a bug...

If we are looking at `ENgetlinkvalue(link_index, EN_INITSETTING)`, can we remove lines [3630-3633](https://github.com/OpenWaterAnalytics/EPANET/blob/dev/src/epanet.c#L3630-L3633)?
```
        if (Link[index].Type == PIPE || Link[index].Type == CVPIPE)
        {
            return EN_getlinkvalue(p, index, EN_ROUGHNESS, value);
        }
```
and return 0 as the default?

**Comment by Mariosmsk** [comment]
@eladsal, I think we need these lines. 

I have tried this and it's ok based on @LRossman 

![image](https://user-images.githubusercontent.com/2945956/74104354-a1b09480-4b5c-11ea-97e8-2d6624cf50b3.png)

**Comment by eladsal** [comment]
> @eladsal, I think we need these lines.

@Mariosmsk I think you are right.

**Comment by stephen-young** [comment]
Is there any way to set the fixed status to `NONE` after the network has been loaded by the API? I.e., is there any risk of this `MISSING` or 0 number to be treated as the actual valve setting?

**Comment by LRossman** [comment]
@stephen-young for a PRV/PSV I think you just need to use` ENsetlinkvalue(linkIndex, EN_INITSETTING, value)` where `value `is your desired setting. When computing hydraulics the code will detect that the setting is no longer `MISSING `and will make the valve obey the setting.

**Comment by eladsal** [comment]
Fixed in #585
